src/components/ui/LoadingMessage.tsx

```diff
@@ -0,0 +1,7 @@
+export const LoadingMessage = () => {
+  return (
+    <div className="min-h-screen flex justify-center items-center">
+      <p className="text-4xl font-ubuntu text-primary-900">Loading...</p>
+    </div>
+  );
```

Comment:
I see that this component is the same as <`HasNoResultsMessage/>` and `<ErrorMessage/>`
You can create one unique component for example `StateMessage` and send the text by prop so we avoid repeating code